### ISSUE TITLE
Using BEST-RQ in downstream tasks

### ISSUE THREAD

**Issue by qocharian** [whettenr/bestrq#3]
Hello, @whettenr ! 
I wanted to discuss the results of my BEST-RQ. 
My recent results are not very good. I did pretraining (600M conformer) on 350-450K hours and fine-tuned on 100K hours. Although the WER initially decreases significantly faster than with random initialization, the final model's WER is worse! I don't know how to interpret these results. 
Perhaps it means that such a model is suitable for low-resource tasks. Maybe you've encountered similar results in your experiments or studies that could explain this. I would be very grateful for your help!

**Comment by whettenr** [comment]
Hi @AraKchrUser,

Would you mind giving the details of the changes that you made when increasing the model size? For example how did you increase the model size? Did you change the training? Batch size? Mask size? How are you doing the downstream task? Is the model frozen?

**Comment by qocharian** [comment]
In fact, I am using my own training pipeline and comparing a randomly initialized model with an SSL-pretrained model. Considering your experience, I wanted to know your opinion and whether you have encountered this personally or in other articles and works. 

I found the following information about pretrain/finetune hours:
1) Multilingual Tasks (from https://arxiv.org/pdf/2202.01855). a) 430K (XLS-R -U) / 10 (MLS) hrs. b) 8M (YT-U) / 1K (MLS) hrs.
2) LibriSpeech (from https://arxiv.org/pdf/2202.01855). 60K / 1K hrs.
3) Multilingual Tasks (from https://arxiv.org/pdf/2404.09841v1). 12M / 1.6M (with pseudo-label).
4) In MOST (from https://arxiv.org/pdf/2303.01037), an order of O(10k) H is claimed for Paired ASR (provided that Unpaired Audio is 12M H).

So, I am using a batch size of 2048 (audio length 3-4 s), the encoder is unfrozen after pretraining (otherwise the results are worse), and CTC/LAS is used for the outputs of the last layers. In my case, the reduction for the conformer is 8, mask is 5%. The model is scaled according to [the article](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/riva/models/speechtotext_en_us_conformer_xl).

**Comment by whettenr** [comment]
In my experiments, I have never experienced a random initialized model performing better. I think the batch size it a pretty good length, the mask is 5% meaning that the mask_prob: 0.05 ?

**Comment by qocharian** [comment]
Yes

**Comment by whettenr** [comment]
What are the results if you try increasing or decreasing this value?

**Comment by qocharian** [comment]
Training on N * 100K hours takes a very long time. How would you suggest validating more efficiently and representatively?
